src/Symfony/Component/Yaml/Inline.php

```diff
@@ -386,12 +386,18 @@ private static function parseSequence(string $sequence, int $flags, int &$i = 0,
                     // the value can be an array if a reference has been resolved to an array var
                     if (\is_string($value) && !$isQuoted && str_contains($value, ': ')) {
                         // embedded mapping?
-                        try {
-                            $pos = 0;
-                            $value = self::parseMapping('{'.$value.'}', $flags, $pos, $references);
-                        } catch (\InvalidArgumentException) {
-                            // no, it's not
-                        }
```

Comment:
The `\InvalidArgumentException` are not caught anymore. Should we keep this catch and eventually remove it in 7.4 if this is identified as unused code?